webapp/src/main/resources/public/js/components/jobs/JobsPage.jsx

```diff
@@ -126,6 +141,13 @@ let JobsPage = createReactClass({
                 <AltContainer store={JobStore} className="mtl mbl" >
                     <JobsView jobType={this.state.jobType} openEditJobModal={this.openEditJobModal} />
                 </AltContainer>
+
+                <JobErrorModal
+                    showModal={this.state.showJobErrorModal}
+                    onErrorModalClosed={this.onCloseJobErrorModal}
+                    errorMessage={this.state.errorMessage || "An error occurred while processing the job."}
+                    title="Error Restoring Job"
```

Comment:
Nit: can we move this prop to be first? 
It's static and gives context for what the intent of this job error modal is